core/controllers/suggestion_test.py

```diff
@@ -65,15 +65,19 @@ def setUp(self):
         self.login(self.EDITOR_EMAIL)
         exploration = (
             self.save_new_linear_exp_with_state_names_and_interactions(
-                self.EXP_ID, self.editor_id, ['State 1', 'State 2', 'State 3'],
-                ['TextInput'], category='Algebra'))
+                self.EXP_ID,
+                self.editor_id,
+                ['State 1', 'State 2', 'State 3', 'State 4'],
+                ['TextInput'],
+                category='Algebra'))
```

Comment:
I think it was fine as above... can you have as many arguments as possible in a single line?

Reply:
Done!